### ISSUE TITLE
Allowing modification or translation of the urls

### ISSUE THREAD

**Issue by avoine** [nephila/djangocms-blog#293]
The BLOG_PERMALINK_URLS system is really convenient for customising the blog post urls.
Maybe we could do the same for the rest of the urls (i.e archive, category, etc)?

What do you think?

**Comment by yakky** [comment]
@avoine if you need to heavily customize URLs, it's probably easier to define your own apphook based on the djangocms-blog one. Eventually I can add an example to the documentation on how to do this (it's just a few lines of code, and they are much easier than providing another layer of setting complexity). What do you think?

**Comment by avoine** [comment]
Yeah, it make sense. I guess in most cases you what to stick with the english url.

I was thinking using ugettext_lazy but it could lead to url collision as the doc say:
https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#django.conf.urls.i18n.i18n_patterns

**Comment by yakky** [comment]
If adding just as example in the documentation is safe to use ``gettext_lazy`` as documented here https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#translating-url-patterns
